docs/document-creator/config-file.md

```diff
@@ -97,7 +124,18 @@ wallet.encrypt("<Enter a password of your choice here>").then((str) => console.l
 
 (_Note: you can watch a tutorial on this [here](https://www.youtube.com/watch?v=z3l9OSVGHH8&feature=youtu.be&t=577) at 7:29 onwards_)
 
-This result can now be entered into the `"wallet"` field in the [config file](#configuration-file).
+This result can now be entered into the `"wallet"` field in the [config file](#encrypted-json-wallet).
+
+#### AWS_KMS Wallet
+
+Ethereum uses Elliptic Curve Digital Signing Algorithm (ECDSA). More specifically, the elliptic curve being used for transaction signing is secp256k1
+
+1) If you don't have a Aws Kms Wallet, you can refer to [documentation](https://luhenning.medium.com/the-dark-side-of-the-elliptic-curve-signing-ethereum-transactions-with-aws-kms-in-javascript-83610d9a6f81) to create a ECC_SECG_P256K1 Key.
+
+2) Connect the Aws Kms Key to an Aws Iam User and enable the signing access control.
+
+3) The IAM credential and KMS Key ID can now be entered into the `"wallet"` field in the [config file](#aws-kms-wallet).
+
 
 ---
```

Comment:
maybe can include a link to `https://docs.tradetrust.io/docs/advanced/aws-kms/overview`

Reply:
i update the documentation link to this overview instead of the external guide link